tool/tsh/tsh.go

```diff
@@ -1548,3 +1548,27 @@ func reissueWithRequests(cf *CLIConf, tc *client.TeleportClient, reqIDs ...strin
 	}
 	return nil
 }
+
+func onApps(cf *CLIConf) {
```

Comment:
@quinqu what's the purpose of `onApps`?

Reply:
I merged from master to update the branch. If you check the diff to master, it doesn't show up.